src/actions/baseAction.ts

```diff
@@ -123,6 +123,10 @@ module Plywood {
       return new ClassFn(value);
     }
 
+    static jsNullSafetyExpressionless(inputJS: string, ifNotNull: (str: string) => string): string {
+      return `(_=${inputJS},(_==null?null:${ifNotNull('_')}))`;
+    }
```

Comment:
you should put this one next to its friend, call them jsNullSafetyUnary and jsNullSafetyBinary